src/extract/bobae.py

```
@@ -0,0 +1,177 @@
+                        datetime.strptime(
+                            dt.find("span", "date").text, "%y.%m.%d %H:%M"
+                        ).timestamp()
+                    )
+
+                    updown = c.find("div", "updownbox")
+                    updown = updown.find_all("a")
+                    comment["comment_likeCount"] = updown[0].text.split(" ")[1]
+                    comment["comment_dislikeCount"] = updown[1].text.split(" ")[1]
+
+                    data["comments"].append(comment)
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)